simulariumio/data_objects/agent_data.py

```diff
@@ -178,6 +186,13 @@ def get_type_ids_and_mapping(self) -> Tuple[np.ndarray, Dict[str, Any]]:
                     last_tid += 1
                     type_id_mapping[tn] = tid
                     type_name_mapping[str(tid)] = {"name": tn}
+                    if (
+                        tn in self.display_data
+                        and not self.display_data[tn].is_default()
+                    ):
+                        type_name_mapping[str(tid)]["geometry"] = dict(
+                            self.display_data[tn]
+                        )
```

Comment:
I don't understand why you're initializing with one default, (SPHERE) but then ignoring the data if only has a default. It seems like you could either have no defaults, and then you just check if the object is empty, or you could just write out default data if none is give so we don't have to to deal with optional data.

Reply:
yeah, I started out without the "FIBER" display_type and this is leftover from that. I just updated it to default to None and only output the display_type if it's been set to something else.